src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1783,6 +1794,23 @@ private static bool IsBlank(FormulaValue arg)
             return false;
         }
 
+        private static FormulaValue IsEmpty(IRContext irContext, FormulaValue[] args)
+        {
+            if (args[0] is BlankValue)
+            {
+                return new BooleanValue(irContext, false);
```

Comment:
>false [](http://example.com/codeflow?start=51&length=5)

If arg[0] is Blank(), shouldn't be TRUE? Usually Blank() means a negative result i.e CountRows(Blank()) is 0. 
 I'm aware this is PA behavior. Just curious on why we decided that Blank() equals to a table with records.

Reply:
Good question - I'll raise it up with the team later

Reply:
Updated it to return true in this case.